packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -44,49 +44,40 @@ contract PseZkVerifier is EssentialContract, IVerifier {
 
     /// @inheritdoc IVerifier
     function verifyProof(
-        uint64, /*blockId*/
-        address prover,
-        bool isContesting,
-        bool usingBlob,
-        bytes32 blobHash,
+        VerifierInput memory input,
         TaikoData.BlockEvidence calldata evidence
     )
         external
         view
     {
         // Do not run proof verification to contest an existing proof
-        if (isContesting) return;
+        if (input.isContesting) return;
 
         PseZkEvmProof memory proof = abi.decode(evidence.proof, (PseZkEvmProof));
 
         bytes32 instance;
-
-        if (usingBlob) {
+        if (input.blobUsed) {
             PointProof memory pf = abi.decode(proof.pointProof, (PointProof));
-            bytes32 x = keccak256(abi.encodePacked(blobHash, pf.txListHash));
+
+            instance = calcInstance({
+                prover: input.prover,
+                evidence: evidence,
+                pointValue: pf.pointValue
+            });
+
             Lib4844.evaluatePoint({
-                blobHash: blobHash,
-                x: uint256(x) % Lib4844.BLS_MODULUS,
+                blobHash: input.blobHash,
+                x: uint256(instance) % Lib4844.BLS_MODULUS,
```

Comment:
please use  `bytes32 x = keccak256(abi.encodePacked(blobHash, pf.txListHash));` as the x value.

Reply:
x is a uint256